FreeRTOS/iot-reference-arm-corstone3xx
# Add support for GCC13

Description
-----------
Added support for GCC13. In addition:

* Fixed GCC12 rwx linker warning
* Removed Stack sealing from GNU linker script

Test Steps
-----------

Validated changes on aws-iot-example/blinky examples and on Corstone-300/310 targets. No regression observed.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
